extra/release_tool.py

```diff
@@ -656,17 +656,42 @@ def do_list_repos(args, optional_too, only_backend):
     assert args.list in cli_types.keys(), "%s is not a valid name type!" % args.list
     type = cli_types[args.list]
 
-    repos = [
-        comp.name
-        for comp in Component.get_components_of_type(
-            type,
-            only_release=(not optional_too),
-            only_non_independent_component=(only_backend),
-        )
-    ]
+    assert args.list_format in ["simple", "table", "json"], (
+        "%s is not a valid name list format!" % args.list_format
+    )
+
+    repos = Component.get_components_of_type(
+        type,
+        only_release=(not optional_too),
+        only_non_independent_component=(only_backend),
+    )
+    repos.sort(key=lambda x: x.name)
+
+    row_format = "| {:>30} | {:>15} |"
```

Comment:
Suggestion:
It'd be nice if the second column was formatted by the length of the longest entry

Reply:
Roger that :+1: